_releases/0.21.0.md

```diff
@@ -0,0 +1,1362 @@
+
+Notable changes
+===============
+
+P2P and network changes
+-----------------------
+
+- The mempool now tracks whether transactions submitted via the wallet or RPCs
+  have been successfully broadcast. Every 10-15 minutes, the node will try to
+  announce unbroadcast transactions until a peer requests it via a `getdata`
+  message or the transaction is removed from the mempool for other reasons.
+  The node will not track the broadcast status of transactions submitted to the
+  node using P2P relay. This version reduces the initial broadcast guarantees
+  for wallet transactions submitted via P2P to a node running the wallet. (#18038)
+
+- The size of the set of transactions that peers have announced and we consider
+  for requests has been reduced from 100000 to 5000 (per peer), and further
+  announcements will be ignored when that limit is reached. If you need to dump
+  (very) large batches of transactions, exceptions can be made for trusted
+  peers using the "relay" network permission. For localhost for example it can
+  be enabled using the command line option `-whitelist=relay@127.0.0.1`.
+  (#19988)
+
+- This release adds support for Tor version 3 hidden services, and rumoring them
+  over the network to other peers using
+  [BIP155](https://github.com/bitcoin/bips/blob/master/bip-0155.mediawiki).
+  Version 2 hidden services are still fully supported by Bitcoin Core, but the
+  Tor network will start
+  [deprecating](https://blog.torproject.org/v2-deprecation-timeline) them in the
+  coming months. (#19954)
+
+- The Tor onion service that is automatically created by setting the
+  `-listenonion` configuration parameter will now be created as a Tor v3 service
+  instead of Tor v2. The private key that was used for Tor v2 (if any) will be
+  left untouched in the `onion_private_key` file in the data directory (see
+  `-datadir`) and can be removed if not needed. Bitcoin Core will no longer
+  attempt to read it. The private key for the Tor v3 service will be saved in a
+  file named `onion_v3_private_key`. To use the deprecated Tor v2 service (not
+  recommended), the `onion_private_key` can be copied over
+  `onion_v3_private_key`, e.g.
+  `cp -f onion_private_key onion_v3_private_key`. (#19954)
+
+- The client writes a file (`anchors.dat`) at shutdown with the network addresses
+  of the node’s two outbound block-relay-only peers (so called "anchors"). The
+  next time the node starts, it reads this file and attempts to reconnect to those
+  same two peers. This prevents an attacker from using node restarts to trigger a
+  complete change in peers, which would be something they could use as part of an
+  eclipse attack. (#17428)
+
+- This release adds support for serving
+  [BIP157](https://github.com/bitcoin/bips/blob/master/bip-0157.mediawiki) compact
+  filters to peers on the network when enabled using
+  `-blockfilterindex=1 -peercfilters=1`. (#16442)
```

Comment:
@laanwj sorry to late.
Using `-peercfilters=1` in 0.21.0 raise a `Invalid parameter` error.  
Isn't `-peerblockfilters=1` correct instead of `-peercfilters=1`?

Reply:
This was fixed in https://github.com/bitcoin/bitcoin/commit/30e664dcce1a9adb9ba9a29e4f0cf809767870dd

Maybe needs backport?

Reply:
Thanks! I have created a PR #742.